sdks/java/io/xml/pom.xml

```diff
@@ -101,7 +101,7 @@
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest-core</artifactId>
-      <version>1.3</version>
+      <scope>test</scope>
```

Comment:
Yes it can be skipped, but I prefer to do it explicitly as we do for example for junit in the parent pom and its children.